src/js/scripts.js

```diff
@@ -0,0 +1,302 @@
+import {
+  appendToDisplay,
+  calculateResult,
+  clearLastElement,
+  clearDisplay,
+} from "./math-utils.js";
+
+import { playSound } from "./drum-utils.js";
+
+import { createBoard, resetBoard } from "./minesweeper-utils.js";
+
+//Application Window Functionality
+
+const calculator = document.getElementById("calculatorApplication");
+const drumKit = document.getElementById("drumkit");
+const notepad = document.getElementById("notepad");
+const minesweeper = document.getElementById("minesweeper");
+
+let activeWindow = null;
+let mousePositions = {};
+let offsets = {};
+let isDown = false;
+
+minesweeper.addEventListener("mousedown", () => {
+  calculator.style.zIndex = 1;
+  drumKit.style.zIndex = 1;
+  notepad.style.zIndex = 1;
+  minesweeper.style.zIndex = 2;
+});
+
+calculator.addEventListener("mousedown", () => {
+  calculator.style.zIndex = 2;
+  drumKit.style.zIndex = 1;
+  notepad.style.zIndex = 1;
+  minesweeper.style.zIndex = 1;
+});
+
+drumKit.addEventListener("mousedown", () => {
+  calculator.style.zIndex = 1;
+  drumKit.style.zIndex = 2;
+  notepad.style.zIndex = 1;
+  minesweeper.style.zIndex = 1;
+});
+
+notepad.addEventListener("mousedown", () => {
+  calculator.style.zIndex = 1;
+  drumKit.style.zIndex = 1;
+  notepad.style.zIndex = 2;
+  minesweeper.style.zIndex = 1;
+});
+
```

Comment:
There's a bit of room here to simplify. The event listener has access to an event object, which should have the target on it. I would assume you can change the target's zIndex to 2 and then change all others to 1. Definitely something to have a play with.